src/index.html

```diff
@@ -11,7 +11,185 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      
+      <fieldset class="vertDistBeetweenGroups">
```

Comment:
```suggestion
      <fieldset class="fieldset">
```
not a big mistake for now, but it's better to set the class name by block meaning, not properties, for example, fieldset / group / form etc

Reply:
OK, thanks! done